1) Add <release> to make Flathub's validator happy
2) remove the icon tag, apparently you can't have that here
--- ring-client-gnome/client-gnome/jami-gnome.appdata.xml	2019-12-09 17:04:40.000000000 +0100
+++ ring-client-gnome/client-gnome/jami-gnome.metainfo.xml	2019-12-12 19:18:46.347919105 +0100
@@ -6,7 +6,9 @@
  <project_license>GPL-3.0+</project_license>
  <name>Jami</name>
  <summary>Privacy oriented voice, video and chat platform.</summary>
- <icon type="stock">jami</icon>
+ <releases>
+  <release version="20200424.2.b47f076" date="2020-04-24" />
+ </releases>
  <description>
   <p>
      Peer-to-peer and end-to-end encrypted audio/video calling, conference, screen sharing, file sharing and messaging application with a distributed architecture that can also be used as a SIP client.